src/components/DataCollector/index.js

```diff
@@ -67,38 +67,25 @@ const createDataCollector = ({ eventManager }) => {
           });
         }
       },
-      applyEvents: {
+      applyHandles: {
         documentationUri: "",
         optionsValidator: serverState => {
           return validateServerState({ options: serverState });
         },
-        run: serverState => {
-          const { request = {}, renderDecisions = false } = serverState;
-
-          const { events: fulfilledEvents = [] } = request.body;
-
-          return Promise.all(
-            fulfilledEvents.map(fulfilledEvent => {
-              const event = eventManager.createEvent();
-
-              const {
-                xdm = {},
-                query = { personalization: { decisionScopes: [] } },
-                data = {}
-              } = fulfilledEvent;
-
-              const decisionScopes = query.personalization.decisionScopes;
+        run: options => {
+          const {
+            renderDecisions = false,
+            decisionScopes,
```

Comment:
Why is decisionScopes necessary? Is there a way we can remove it so that the customer doesn't have to duplicate the data from the server to the client call?